src/Data/Singletons/Prelude/Traversable.hs

```diff
@@ -204,7 +205,13 @@ $(singletonsOnly [d|
   deriving instance Traversable NonEmpty
   deriving instance Traversable (Either a)
   deriving instance Traversable ((,) a)
-  deriving instance Traversable (Const m)
```

Comment:
Should this have been deleted?

Reply:
Sigh, that definitely wasn't intended. I've added this back in ad4a8e90f342183b685ce3bd935ea5d1bf6ad6a3.